services/QuillLMS/spec/system/activity_pack_assignment_spec.rb

```diff
@@ -69,6 +69,9 @@
     # the following line checks for the presence of an element that only loads once activity data has loaded, ensuring that the assign click will be successful
     first('.review-activities-data-table-section').click
     click_on 'Assign pack to classes'
+
+    # TODO: Fix bug in which you can click too fast and break this flow (remove sleep when that is done)
```

Comment:
I believe this is an unrelated to my code failure. I'll create a card to fix this.